hasjob/models/domain.py

```diff
@@ -52,6 +53,15 @@ def use_title(self):
     def has_profile(self):
         return bool(self.title and self.description)
 
+    @cached_property
+    def get_logo_url(self):
```

Comment:
You don't need `get_` in the name if it's a property, since `get` implies that it needs to be called. Naming this `logo_url` should suffice.